docs/markdown/Getting Started/getting-started/manual-installation.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Manual and advanced installation"
+slug: "manual-installation"
+hidden: false
+createdAt: "2022-07-12T00:00:00.000Z"
+updatedAt: "2022-07-12T00:00:00.000Z"
+---
+
+If you want to make use of a non-standard Pants installation, for example, using a local source installation of Pants, follow these steps.
+
+
+Manual installation
+-------------------
+
+Pants is invoked via a launch script named `./pants` , saved at the root of the repository. This script will install Pants and handle upgrades.
+
+First, set up a minimal `pants.toml` config file to instruct the script to download the latest 2.13 release:
+
+```bash
+printf '[GLOBAL]\npants_version = "2.13.0.dev5"\n' > pants.toml
```

Comment:
We're gonna struggle to keep this up-to-date, as we always have. Thoughts on having the user manually look up the latest version by searching PyPI?

Reply:
I think that's a good follow-up. I have a snippet that uses `jq` that will do this automatically, but I think that's going to a source of bikeshedding